[2025-05-06T17:22:04.536+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql scheduled__2025-05-06T17:21:00+00:00 [queued]>
[2025-05-06T17:22:04.544+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql scheduled__2025-05-06T17:21:00+00:00 [queued]>
[2025-05-06T17:22:04.544+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-06T17:22:04.558+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): sync_and_load_mysql> on 2025-05-06 17:21:00+00:00
[2025-05-06T17:22:04.563+0000] {standard_task_runner.py:60} INFO - Started process 37246 to run task
[2025-05-06T17:22:04.566+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mymind_mongo_to_mysql_etl', 'sync_and_load_mysql', 'scheduled__2025-05-06T17:21:00+00:00', '--job-id', '35525', '--raw', '--subdir', 'DAGS_FOLDER/mymind_etl_dag.py', '--cfg-path', '/tmp/tmpflf9q2ej']
[2025-05-06T17:22:04.567+0000] {standard_task_runner.py:88} INFO - Job 35525: Subtask sync_and_load_mysql
[2025-05-06T17:22:04.614+0000] {task_command.py:423} INFO - Running <TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql scheduled__2025-05-06T17:21:00+00:00 [running]> on host 52beca4b988b
[2025-05-06T17:22:04.705+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mymind_mongo_to_mysql_etl' AIRFLOW_CTX_TASK_ID='sync_and_load_mysql' AIRFLOW_CTX_EXECUTION_DATE='2025-05-06T17:21:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-06T17:21:00+00:00'
[2025-05-06T17:22:04.706+0000] {mymind_etl_dag.py:301} INFO - Iniciando sincronización y carga a MySQL: 6 usuarios, 9 transcripciones.
[2025-05-06T17:22:04.714+0000] {base.py:83} INFO - Using connection ID 'mysql_mymind_dw' for task execution.
[2025-05-06T17:22:04.722+0000] {mymind_etl_dag.py:310} INFO - Obteniendo IDs actuales desde MySQL...
[2025-05-06T17:22:04.722+0000] {mymind_etl_dag.py:321} INFO - Usuarios a eliminar: 0
[2025-05-06T17:22:04.723+0000] {mymind_etl_dag.py:322} INFO - Transcripciones a eliminar: 0
[2025-05-06T17:22:04.723+0000] {mymind_etl_dag.py:355} INFO - Cargando 6 usuarios en la tabla users...
[2025-05-06T17:22:04.724+0000] {mymind_etl_dag.py:380} INFO - Usuarios cargados exitosamente.
[2025-05-06T17:22:04.724+0000] {mymind_etl_dag.py:393} INFO - Cargando 9 transcripciones en la tabla transcriptions...
[2025-05-06T17:22:04.725+0000] {mymind_etl_dag.py:423} INFO - Transcripciones cargadas exitosamente.
[2025-05-06T17:22:04.725+0000] {mymind_etl_dag.py:430} INFO - Sincronización y carga a MySQL completada con éxito.
[2025-05-06T17:22:04.726+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-06T17:22:04.734+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=mymind_mongo_to_mysql_etl, task_id=sync_and_load_mysql, execution_date=20250506T172100, start_date=20250506T172204, end_date=20250506T172204
[2025-05-06T17:22:04.778+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-06T17:22:04.795+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
